data/mods/Aftershock/mobs/robots.json

```diff
@@ -732,32 +732,43 @@
     ]
   },
   {
-    "id": "mon_advbot_plasma",
+    "id": "mon_wraitheon_irradiant",
     "type": "MONSTER",
     "copy-from": "mon_advbot_base",
-    "name": "plasma-ejecting robot",
-    "description": "An advanced robot still functioning due to its internal fusion core.  This model is furnished with a powerful plasma-ejector.",
-    "diff": 20,
-    "color": "magenta",
+    "name": "Wraitheon Irradiant",
+    "description": "An autonomous anti-armor weapons plataform, mounting a 5kg plasma railgun over an insectoid chasis.  The Irradiant designation excels at ambush and midrage range combat, where it can leverage its superior agility and reduced sensor profile to succesfully engage technologically advanced foes.",
```

Comment:
```suggestion
    "description": "An autonomous anti-armor weapons plataform, mounting a 5kg plasma railgun over an insectoid chasis.  The Irradiant designation excels at ambush and midrange combat, where it can leverage its superior agility and reduced sensor profile to successfully engage technologically advanced foes.",
```

Reply:
Superseded by the suggestion below, thank you.